### ISSUE TITLE
[Bug]: When using babel with preset-env I get errors in Chrome, IE11, Edge and FF

### ISSUE THREAD

**Issue by YonatanKra** [babel/babel#7113]
**Bug report**

### Input Code
Here's a branch with the issue:
https://github.com/YonatanKra/YOPF/tree/babel-with-dynamic-import

### Babel/Babylon Configuration (.babelrc, package.json, cli command)
In branch: https://github.com/YonatanKra/YOPF/blob/babel-with-dynamic-import/webpack.config.js

### Expected Behavior
I expect the app to load with no error

### Current Behavior
When using babel with preset-env in IE11, I get an error: `Object.getOwnPropertyDescriptor: argument is not an Object`
Chrome/Edge/FF show a different error: `Uncaught ReferenceError: regeneratorRuntime is not defined`

### Context
The app works fine in Chrome/Edge/FF without babel or when using `babel-loader` without the `present-env`.  In order to support older browsers, I've tried to add `babel` to the project and had these issues.

### Your Environment

| software         | version(s)
| ---------------- | -------
| Babel            |  ^7.0.0-beta.36
| @babel/preset-env | ^7.0.0-beta.36
| node             |  6.9.1
| npm              |  3.10.8
| Operating System |  Windows 10

**Comment by nicolo-ribaudo** [comment]
I don't think that this causes the problem, but you should use babel/plugin-syntax-dynamic-import, not babel-plugin-syntax-dynamic-import
